diff --git a/cmake/GoogleCloudCppFeatures.cmake b/cmake/GoogleCloudCppFeatures.cmake
index cf2d9bc8e..9060937f4 100644
--- a/cmake/GoogleCloudCppFeatures.cmake
+++ b/cmake/GoogleCloudCppFeatures.cmake
@@ -42,7 +42,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "apigeeconnect"
     "appengine"
     "artifactregistry"
-    "asset"
+    # "asset" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8145
     "assuredworkloads"
     "automl"
     "baremetalsolution"
@@ -52,7 +52,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "billing"
     "binaryauthorization"
     "certificatemanager"
-    "channel"
+    # "channel" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8125
     "cloudbuild"
     "composer"
     "connectors"
@@ -115,7 +115,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "spanner"
     "speech"
     "storage"
-    "storagetransfer"
+    # "storagetransfer" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8785
     "talent"
     "tasks"
     "texttospeech"
