diff --git a/cmake/GoogleCloudCppFeatures.cmake b/cmake/GoogleCloudCppFeatures.cmake
index 2293eca6d..f046a7b34 100644
--- a/cmake/GoogleCloudCppFeatures.cmake
+++ b/cmake/GoogleCloudCppFeatures.cmake
@@ -52,7 +52,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "billing"
     "binaryauthorization"
     "certificatemanager"
-    "channel"
+#    "channel"
     "cloudbuild"
     "composer"
     "connectors"
