From 06db3d22c2a6466d690cbc05596627c763e3405e Mon Sep 17 00:00:00 2001
From: Carlos O'Ryan <coryan@google.com>
Date: Tue, 7 Feb 2023 01:35:26 +0000
Subject: [PATCH 1/2] disable libraries

---
 cmake/GoogleCloudCppFeatures.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/GoogleCloudCppFeatures.cmake b/cmake/GoogleCloudCppFeatures.cmake
index 7f7de7b4b3..c42e3bf4a0 100644
--- a/cmake/GoogleCloudCppFeatures.cmake
+++ b/cmake/GoogleCloudCppFeatures.cmake
@@ -44,7 +44,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "apigeeconnect"
     "appengine"
     "artifactregistry"
-    "asset"
+    # "asset" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8145
     "assuredworkloads"
     "automl"
     "baremetalsolution"
@@ -55,7 +55,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "billing"
     "binaryauthorization"
     "certificatemanager"
-    "channel"
+    # "channel" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8125
     "cloudbuild"
     "composer"
     "confidentialcomputing"
@@ -125,7 +125,7 @@ set(GOOGLE_CLOUD_CPP_GA_LIBRARIES
     "speech"
     "storage"
     "storageinsights"
-    "storagetransfer"
+    # "storagetransfer" -- needs newer Protobuf https://github.com/googleapis/google-cloud-cpp/issues/8785
     "support"
     "talent"
     "tasks"
