From d9f5e22ce65f427a42d5988fa7227d2dc257b7db Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Wed, 31 Aug 2022 19:00:34 +0200
Subject: [PATCH 3/3] do not try to build external/googleapis as shared

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 028802691d..59fb481451 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -207,7 +207,10 @@ host environment.
             "find_dependency(Protobuf)")
         find_package(Protobuf REQUIRED)
     endif ()
+    # do not try to build googleapis shared on windows
+    set(BUILD_SHARED_LIBS OFF)
     add_subdirectory(external/googleapis)
+    set(BUILD_SHARED_LIBS ON)
 endif ()
 
 # Enable testing in this directory so we can do a top-level `make test`. This
